src/wh_server.c

```diff
@@ -54,7 +621,7 @@ int wh_Server_Init(whServerContext* server, whServerConfig* config)
 static int _wh_Server_HandleCommRequest(whServerContext* server,
         uint16_t magic, uint16_t action, uint16_t seq,
         uint16_t req_size, const void* req_packet,
-        uint16_t *out_resp_size, void* resp_packet)
```

Comment:
As a style point, I'll normally leave the * attached to an "output" parameter.  Ok to change if this is confusing.

Reply:
`(void*)myVal` or `(void *)myVal`? makes no sense putting part of the type on the variable name